packages/@coorpacademy-components/src/molecule/teams-popin/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  NovaCompositionCoorpacademyReload as ReloadIcon,
+  NovaCompositionCoorpacademyLoginFaild as LoginFailedIcon,
+  NovaCompositionCoorpacademyAddressError as AddreessErrorIcon
+} from '@coorpacademy/nova-icons';
+import style from './style.css';
+
+const Icon = props => {
+  const {type} = props;
+  switch (type) {
+    case 'reload':
+      return (
+        <div className={style.iconContainer}>
+          <ReloadIcon className={style.icon} />
+        </div>
+      );
+    case 'wrong':
+      return (
+        <div className={style.iconContainer}>
+          <ReloadIcon className={style.icon} />
+        </div>
+      );
+    case 'addressError':
+      return (
+        <div className={style.iconContainer}>
+          <AddreessErrorIcon className={style.icon} />
+        </div>
+      );
+    case 'loginFailed':
+      return (
+        <div className={style.iconContainer}>
+          <LoginFailedIcon className={style.icon} />
+        </div>
+      );
+    default:
+      return null;
+  }
+};
+
+const TeamsPopin = props => {
+  const {header, content, buttonLabel, onButtonClick, type} = props;
+  return (
+    <div className={style.wrapmessage}>
+      <div className={style.disable}>
+        <div className={style.popin}>
+          <Icon type={type} />
```

Comment:
const IconType = ICONS[type];
```suggestion
          <div className={style.iconContainer}>
               <IconType className={style.icon} />
          </div>
```